### DISCUSSION TITLE
Löschen von Mitgliedern mit Buchungen

### DISCUSSION THREAD

**Discussion by mbmueller** [openjverein/jverein#170] [Ideas]
Ich würde gerne meine Mitglieder DSGVO-konform löschen, die geht allerdings nicht, da diesem noch Buchungen zugeordnet sind, alle älter als 10 Jahre, die in einem Abschluss erfasst wurden. Habt ihr da einen Workaround? Im Idealfall sogar ein Script oder so?

**Comment by JohannMaierhofer** [comment]
Ich habe mir einmal die Datenbank angeschaut.
Sie ist so konfiguriert, dass beim Löschen des Mitglieds automatisch Einträge für das Mitglied in den Tabellen Eigenschaften, Arbeitseinsatz, Lehrgang, Mailempfänger, Mitgliedsfoto, Mitgliedskonto (Sollbuchung), und Zusatzfelder gelöscht werden. Gibt es also nur dort Einträge kann man das Mitglied löschen.
Folgende Tabellen sind so konfiguriert, dass man Das Mitglied nicht löschen kann wenn dort Einträge für das Mitglied vorhanden sind. Es sind die Tabellen Sekundärbeitragsgruppe, Lastschrift, Mitgliedsdokument, Spendenbescheinigung, Wiedervorlage, Zusatzabbuchung.
Ich weiß jetzt nicht, warum das so konfiguriert ist. Man könnte doch auch diese Einträge automatisch löschen evtl. nach einer Sicherheitsabfrage.

**Comment by mbmueller** [comment]
Wird durch #224  implementiert.
